app/frontend/wallet/shelley/shelley-bitbox02-crypto-provider.ts

```diff
@@ -299,16 +296,31 @@ const ShelleyBitBox02CryptoProvider = async ({
       })
     }
 
+    const ttl = txAux.ttl
+    if (ttl == null) {
+      // not done via isFeatureSupported() because it would be too much hassle
+      // and we suppose the new new FW will support that together with missing multiasset
+      // support which once added, we'd bump the minimal suported version to that
+      //
+      // TODO: bump minimum supported version of Ledger/Trezor as well to the version
```

Comment:
not doing now, because it would be too big of a change, I'd do it as some bigger, systematic cleanup at some point